src/components/TypeAhead.svelte

```diff
@@ -25,50 +25,68 @@
     } catch (ignore) {
       results = ["Cannot retrieve results."];
     }
-    var ul = document.getElementById("test");
-    ul.onclick = function(event) {
-      var target = getEventTarget(event);
-      value = target.innerHTML;
-    };
   };
+  function selectId(event) {
+    value = event.detail.text;
+    showResults = false;
+  }
+  function refresh() {
+    showResults = true;
+    value = "";
+  }
```

Comment:
The button is neat but I don't actually think this is the proper UI flow. Instead, on keyup, you should set showResults back to true. You also don't need to clear the value.

The flow should work like this:
* user starts typing into input
* user selects item from input (or hits Enter for the first one in the list)
* the UI changes in a manner that displays that the user has selected an item, and the list goes away
* when the user starts typing in the input again, deselect the item and go back to showing results from a list

We're faking a drop-down list, essentially.